Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -214,65 +221,59 @@ class TabTrayViewController: UIViewController, Themeable {
         // We expose the tab tray feature whenever it's going to be seen by the user
         nimbus.features.tabTrayFeature.recordExposure()
 
-        if shouldUseiPadSetup() {
-            navigationController?.isToolbarHidden = true
-        } else {
-            navigationController?.isToolbarHidden = false
-            updateToolbarItems(forSyncTabs: viewModel.profile.hasSyncableAccount())
-        }
+        updateLayout()
     }
 
     private func viewSetup() {
         viewModel.syncedTabsController.remotePanelDelegate = self
 
-        if shouldUseiPadSetup() {
-            iPadViewSetup()
-        } else {
-            iPhoneViewSetup()
+        // iPad setup
+        navigationItem.titleView = segmentedControlIpad
+
+        if let titleView = navigationItem.titleView {
+            titleWidthConstraint = titleView.widthAnchor.constraint(equalToConstant: UX.NavigationMenu.width)
+            titleWidthConstraint?.isActive = true
         }
 
+        // iPhone setup
+        view.addSubview(navigationToolbar)
+        navigationToolbar.setItems([UIBarButtonItem(customView: segmentedControlIphone)], animated: false)
+
+        NSLayoutConstraint.activate([
+            navigationToolbar.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            navigationToolbar.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            navigationToolbar.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+
+            segmentedControlIphone.widthAnchor.constraint(lessThanOrEqualToConstant: UX.NavigationMenu.width),
+            segmentedControlIphone.heightAnchor.constraint(equalToConstant: UX.NavigationMenu.height)
+        ])
+
         showPanel(viewModel.tabTrayView)
     }
 
     func updatePrivateUIState() {
         UserDefaults.standard.set(viewModel.tabManager.selectedTab?.isPrivate ?? false, forKey: "wasLastSessionPrivate")
     }
 
-    fileprivate func iPadViewSetup() {
-        navigationItem.leftBarButtonItem = deleteButton
-        navigationItem.titleView = navigationMenu
-        navigationItem.rightBarButtonItems = [doneButton, fixedSpace, newTabButton]
-
-        navigationItem.titleView?.snp.makeConstraints { make in
-            make.width.equalTo(343)
+    fileprivate func updateTitle() {
```

Comment:
```suggestion
    private func updateTitle() {
```
nit: replace fileprivate